src/dune_pkg/local_package.mli

```diff
@@ -48,6 +56,8 @@ module For_solver : sig
     ; depopts : Package_dependency.t list
     ; conflict_class : Package_name.t list
     ; pins : pins
+    ; build : OpamTypes.command list
```

Comment:
Is it necessary to add the commands to this type? I wouldn't have thought the solver needs that information.

Reply:
Yes it is. Otherwise, the information is lost later because of this:
```ocaml
let resolved_package =
        let local_package = Local_package.of_package pkg in
        let opam_file =
          local_package
          |> Local_package.for_solver
          |> Local_package.For_solver.to_opam_file
          (*                           ^^^^^^ This needs the build and install to not lost the information *)
          |> OpamFile.OPAM.with_url (OpamFile.URL.create (snd package.url))
        in
```